Makefile

```diff
@@ -13,6 +13,8 @@ run-docker:
 	docker run -it -v `pwd`:/terraform-validator -v ${GOOGLE_APPLICATION_CREDENTIALS}:/terraform-validator/credentials.json --entrypoint=/bin/bash --env TEST_PROJECT=${PROJECT_ID} --env TEST_CREDENTIALS=./credentials.json terraform-validator;
 
 test-integration:
+	go version
```

Comment:
I wasn't able to figure out what go version and terraform version were being used in the tests otherwise. It ended up not being relevant in this case, but I would rather have those values easily available for the future.